src/ctap/command.rs

```diff
@@ -601,7 +601,7 @@ mod test {
             0x05 => cbor_array![],
             0x08 => vec![0x12, 0x34],
             0x09 => 1,
-            0x0A => true,
+            0x0A => 2,
```

Comment:
I'm assuming this is within the same draft, so there's no compatibility concerns.

Reply:
This actually didn't change within the specification, it's just that I assigned the wrong type to the stub.